packages/lib/survey/utils.ts

```
@@ -100,14 +100,18 @@
+    const { object } = await generateObject({
+      model: llmModel,
+      schema: z.object({
+        insightsEnabled: z.boolean(),
+      }),
+      prompt: `We extract insights (e.g. feature requests, complaints, other) from survey questions. Can we find them in this question?: ${question.headline.default}`,
+      experimental_telemetry: { isEnabled: true },
+    });
+
+    return object.insightsEnabled;
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)